### DISCUSSION TITLE
Welcome to coding-youtubers Discussions!

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/coding-youtubers#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Abhinavkar** [comment]
Should we make a page for this edition" what are the best content you will get on youtube " this will the motive for the webpage 
we can make it a challenge by creating the webpage using html css js on;y so that new contributers will learn and get a chance also after then we ccan it tough one by implementing in other languages 
what are your views guys @KendallDoesCoding @vikasganiga05

**Reply by KendallDoesCoding** [reply]
I can't understand what you mean.
